src/org/zaproxy/zap/extension/domxss/TestDomXSS.java

```diff
@@ -513,6 +522,7 @@ public void scan() {
 		} finally {
 			this.returnFirefoxDriver(fxDriver);
 		}
+		super.scan();
```

Comment:
This could be called in the above try block (after the for loop), then in `scan(HttpMessage, String, String)` the try/catch and try/finally blocks could be removed.